### PR TITLE
(Fix, Refactor) Add support of a new version of MetaMask

### PR DESCRIPTION
**Pull request by varasev**
- (Mandatory) Description

    This PR solves the issue https://github.com/poanetwork/poa-dapps-voting/issues/179 and makes some refactoring of code. MetaMask v4.14.0 (and above) will display the next confirmation message when a user opens DApp for the first time after `November 2nd, 2018`:

    image
    
    If the user clicks on `Approve`, the DApp will load and continue working. Otherwise (in the case of `Reject`), DApp will show the next alert and stop execution:

    image

    If the user clicks on `Approve`, MetaMask will cache user's choice and won't ask the user again next time until he/ she clears approval data in MetaMask's settings.

    The custom build of MetaMask for testing can be downloaded here: https://github.com/MetaMask/metamask-extension/pull/4703#issuecomment-427062852.

    These changes are backward compatible, so the new code will work fine with old DApp browsers that don't support `ethereum.enable()` feature yet.

- (Mandatory) What is it: (Fix), (Feature), or (Refactor)
(Fix, Refactor)

### CODE REVIEW

**Review by vbaranov — COMMENTED** [review]
See https://github.com/poanetwork/poa-dapps-validators/pull/88#issue-222920534

File: src/constants.js [hunk 1]
```diff
@@ -24,10 +24,34 @@ constants.minBallotDurationInDays = 2
...
 constants.getTransactionReceiptInterval = 5000
-constants.NETID_SOKOL = '77'
-constants.NETID_CORE = '99'
-constants.NETID_DAI_TEST = '79'
-constants.NETID_DAI = '100'
+
+constants.NETWORKS = {
+  '77': {
+    NAME: 'Sokol',
+    RPC: 'https://sokol.poa.network',
```
Comment (vbaranov) [inline src/constants.js]: @varasev where do we use RPC endpoints of chains?

Comment (varasev) [reply src/constants.js]: @vbaranov It's just a copy-paste from Validators DApp where they are used. Voting DApp doesn't use them. Forgot to remove. I'll fix this.

Comment (varasev) [reply src/constants.js]: @vbaranov Removed RPCs in https://github.com/poanetwork/poa-dapps-voting/pull/181/commits/5b3d9740a758adb80511821c85002129aac32bd6

**Review by vbaranov — APPROVED** [review]

**Pull request merged**